source/includes/compatibility-table-legend.rst

```diff
@@ -0,0 +1,18 @@
+Compatibility Table Legend
+++++++++++++++++++++++++++
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility
+
+   * - Icon
+     - Explanation
+
+   * - ✓
+     - All features are supported.
+   * - ⊛
+     - The Driver version will work with the MongoDB version, but not all
+       new MongoDB features are supported.
+   * - No mark
+     - This Driver version is not tested with the MongoDB version.
```

Comment:
Nitpick/Suggestion: Consistent phrasing - the first bullet uses the phrase "The Driver" and the second one uses "This Driver". I'd suggest making the two match